third_party/move/move-vm/test-utils/src/storage.rs

```diff
@@ -105,6 +105,12 @@ impl<'a, 'b, S: ModuleResolver> ModuleResolver for DeltaStorage<'a, 'b, S> {
     }
 }
 
+fn get_bytes_and_size(buf: Option<BytesWithLayout>) -> Result<(Option<Bytes>, usize)> {
+    let buf_bytes = buf.as_ref().map(|(bytes, _)| bytes);
```

Comment:
Why as_ref? the value is owned?

Reply:
I'm actually not sure why. It gave an error when not using as_ref. So, I just used as_ref to clear the error.